misk-testing/src/main/kotlin/misk/logging/RealLogCollector.kt

```diff
@@ -10,37 +10,12 @@ import javax.inject.Singleton
 import kotlin.reflect.KClass
 
 @Singleton
-internal class RealLogCollector @Inject constructor() :
-  AbstractIdleService(),
-  LogCollector,
+internal class RealLogCollector @Inject constructor(
+  private val wispQueuedLogCollector: WispQueuedLogCollector
+) : AbstractIdleService(),
+  LogCollector by wispQueuedLogCollector,
   LogCollectorService {
```

Comment:
I think ideally `LogCollectorService` could be delegated to the `WispQueuedLogCollector`, but I gues wisp-logging-testing doesn't want to depend on Guava?